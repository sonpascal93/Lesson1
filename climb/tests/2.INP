20
-9 4 6 -9 -3 3 -1 2 -6 5 -7 5 5 4 -2 -5 2 9 6 2 