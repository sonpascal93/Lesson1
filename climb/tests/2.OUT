11
1 4 5 7 8 10 12 13 14 15 16 